core/cbor/cbor.go

```diff
@@ -24,18 +24,7 @@ func ParseDietCBOR(b []byte) (models.JSON, error) {
 		return models.JSON{}, err
 	}
 
-	coerced, err := CoerceInterfaceMapToStringMap(m)
-	if err != nil {
-		return models.JSON{}, err
-	}
-
-	jsb, err := json.Marshal(coerced)
-	if err != nil {
-		return models.JSON{}, err
-	}
-
-	var js models.JSON
-	return js, json.Unmarshal(jsb, &js)
```

Comment:
This Marshal/Unmarshal converted []byte to base64 and for big ints it truncated long numbers, not sure the latter was intentional or not.